media/contentCreator/scaffoldAnsibleProjectPageStyle.css

```diff
@@ -0,0 +1,124 @@
+body {
```

Comment:
I guess this file needs to be added to the `.vscodeignore` file for .vsix packaging